.github/workflows/continuous-integration.yaml

```diff
@@ -29,6 +29,15 @@ jobs:
           distribution: 'zulu'
           java-version: '8'
 
+      - name: Cache dependencies
```

Comment:
Doesn't this already happen later on?

Reply:
It happens for each set of modules separately. I added this as I noticed the Javadoc is doing a build of the code but didn't have a cache setup before it runs, which means it needs to download every time. Admittedly, this only saves 30 sec to 1 min, but it is good practice.